src/tflite_cpu.cpp

```diff
@@ -271,40 +227,27 @@ class MLModelServiceTFLite : public vsdk::MLModelService,
 
     struct metadata metadata(const vsdk::ProtoStruct& extra) final {
         // Just return a copy of our metadata from leased state.
-        return lease_state_()->metadata;
+        std::lock_guard<std::mutex> lock(state_lock_);
+        check_stopped_inlock_();
+        return state_->metadata;
     }
 
    private:
-    struct state;
+    struct state_;
 
     void check_stopped_inlock_() const {
-        if (stopped_) {
+        if (!state_) {
```

Comment:
If we don't have anything in `state_` then either we were 1) stopped, or 2) failed reconfigure which is equivalent to being stopped.